src/components/ElementSearchDialog/element-search-dialog.js

```diff
@@ -66,10 +68,16 @@ const ElementSearchDialog = (props) => {
                         setExpanded(false);
                     }}
                     fullWidth
-                    onInputChange={(_event, value) =>
-                        handleSearchTermChange(value)
-                    }
-                    onChange={(_event, newValue) => onSelectionChange(newValue)}
+                    onInputChange={(_event, value) => {
+                        if (!searchTermDisabled) {
+                            handleSearchTermChange(value);
+                        }
+                    }}
+                    onChange={(_event, newValue) => {
+                        if (!searchTermDisabled && newValue) {
```

Comment:
Really needed test ?

Reply:
Yes. otherwise, there is a crash when the user deletes all characters in the textfield (newValue is null)